src/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const Calculator = () => (
+
+  <table className="table">
+    <tbody>
+      <tr>
+        <td>Ac</td>
+        <td>+/-</td>
+        <td>%</td>
+        <td>/</td>
+      </tr>
+      <tr>
```

Comment:
- [ ] As noted by the previous reviewer, there should be at least one component in this component, you can achieve this by extracting some of the code to another file and then importing it here.